Formula/g/gossip.rb

```diff
@@ -0,0 +1,23 @@
+class Gossip < Formula
+  desc "Desktop client for Nostr written in Rust"
+  homepage "https://github.com/mikedilger/gossip"
+  url "https://github.com/mikedilger/gossip.git",
+      tag:      "v0.8.0",
+      revision: "c9bb3704e94e105d7cf97de67e89ccf7f01ef69b"
+  license "MIT"
+  head "https://github.com/mikedilger/gossip.git", branch: "master"
+
+  depends_on "rust" => :build
+  depends_on :macos # linux build requires additional runtime dependencies
```

Comment:
Then you should add them

Reply:
I don't know what they all are exactly or if they are even available from homebrew. I was not able to get it working on Linux without installing additional packages with the native package manager.

Reply:
If the software is supported on all homebrew platforms we should build it for all platforms.

Reply:
Resolved in 47f5464c60909507f530cabe8faf415bca5dd3f3